packages-next/keystone/src/lib/createExpressServer.ts

```diff
@@ -82,6 +82,17 @@ export const createExpressServer = async (
   const { graphQLSchema, createContext } = system;
   addApolloServer({ server, graphQLSchema, createContext, sessionImplementation });
 
+  const apps = ([...system.keystone.registeredTypes])
```

Comment:
I had to copy this from keystone prepare code which does not seems to be running anymore in new interface. Have to figure out how to optimize this. I am sure this is not the best way